lib/perl/Genome/Model/Event/Build/ReferenceAlignment/AlignReads.pm

```diff
@@ -488,7 +489,7 @@ sub _process_and_link_alignments_to_build {
     }
 
     for my $alignment (@alignments) {
-        my $link = $alignment->add_user(user => $build, label => 'uses');
+        my $link = $alignment->user(user => $build, label => 'uses');
```

Comment:
The problem is that `add_user` can be called from both `shortcut` and `execute`.  In this case `shortcut` failed in the flow wrapper after `commit` added the user.  Thereafter `execute` was run which blindly added the user again.

The other `add_user` calls are also susceptible to this problem.  I should change them all.  This is the only one that has come up as a problem in practice (since I did the massive cleanup) so it was the first to be changed.